lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -110,7 +111,7 @@ def display_peek(self, dataset):
 
 class PlyAscii(Ply, data.Text):
 
-    file_ext = "plyascii"
+    file_ext = "ply"
```

Comment:
@nsoranzo is correct - there are binary and ascii types of the ply datatype, so we need to differentiate them.